src/Pages/AdminTaskTable/AdminTaskTable.tsx

```diff
@@ -0,0 +1,274 @@
+    justifyContent: 'space-between',
+    height: 'fit-content',
+  },
+  invisible: {
+    opacity: 0,
+  },
+});
+
+export const formatDate = (date: string): string => dayjs(date).format('DD MMM YYYY HH:mm UTCZ');
+
+const AdminTaskTable = () => {
+  const classes = useStyles();
+  const storedPerPage = Number(localStorage.getItem('perPage')) || 20;
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(storedPerPage);
+  const [activeSortIndex, setActiveSortIndex] = useState<number>(0);
+  const [activeSortDirection, setActiveSortDirection] = useState<'asc' | 'desc'>('asc');
+  const [payloadOpen, setPayloadOpen] = useState(false);
+  const { fetchAdminTask, isLoading: isFetchingAdminTask } = useFetchAdminTaskQuery();
+  const [adminTask, setAdminTask] = useState<AdminTask | null>(null);
+
+  const [filterData, setFilterData] = useState<AdminTaskFilterData>({
+    statuses: [],
+    accountId: '',
+    orgId: '',
+  });
+
+  const clearFilters = () => setFilterData({ statuses: [], accountId: '', orgId: '' });
+
+  const notFiltered =
+    filterData.statuses.length === 0 && filterData.accountId === '' && filterData.orgId === '';
+
+  const columnSortAttributes = [
+    'org_id',
+    'account_id',
+    'typename',
+    'queued_at',
+    'started_at',
+    'finished_at',
+    'status',
+  ];
+
+  const sortString = (): string =>
+    columnSortAttributes[activeSortIndex] + ':' + activeSortDirection;
+
+  const {
+    isLoading,
+    error,
+    isError,
+    isFetching,
+    data = { data: [], meta: { count: 0, limit: 20, offset: 0 } },
+  } = useAdminTaskListQuery(page, perPage, filterData, sortString());
+
+  const actionTakingPlace = isFetching;
+
+  const onSetPage: OnSetPage = (_, newPage) => setPage(newPage);
+
+  const onPerPageSelect: OnPerPageSelect = (_, newPerPage, newPage) => {
+    // Save this value through page refresh for use on next reload
+    localStorage.setItem('perPage', newPerPage.toString());
+    setPerPage(newPerPage);
+    setPage(newPage);
+  };
+
+  const sortParams = (columnIndex: number): ThProps['sort'] => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection,
+      defaultDirection: 'asc', // starting sort direction when first sorting a column. Defaults to 'asc'
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+    },
+    columnIndex,
+  });
+
+  const columnHeaders = ['Org ID', 'Account ID', 'Type', 'Queued At', 'Status'];
+
+  // Error is caught in the wrapper component
+  if (isError) throw error;
+
+  const {
+    data: adminTaskList = [],
+    meta: { count = 0 },
+  } = data;
+
+  const countIsZero = count === 0;
+
+  if (countIsZero && notFiltered && !isLoading)
+    return (
+      <Bullseye data-ouia-safe={!actionTakingPlace} data-ouia-component-id='admin_task_list_page'>
+        <EmptyTableState notFiltered={notFiltered} clearFilters={clearFilters} />
```

Comment:
We now have a few of these, we may want to create a reusable component for this (ie title/paragraph variables and combine the few we have).